build.gradle

```diff
@@ -4,11 +4,15 @@ plugins {
     id 'java-library'
     id 'maven-publish'
 
-    id 'com.diffplug.spotless' version '5.10.2'
-    id 'com.github.ben-manes.versions' version '0.33.0'
+    id 'com.diffplug.spotless' version '5.12.4'
```

Comment:
If there are new rules or guidelines (such as avoiding `+`), then we should add them to the repo documentation.